constance/templates/admin/constance/change_list.html

```diff
@@ -15,6 +15,7 @@
 {{ block.super }}
 {{ media.js }}
 <script type="text/javascript" src="{% static 'admin/js/constance.js' %}"></script>
+<script type="text/javascript" src="{% static '/admin/js/collapse.js' %}"></script>
```

Comment:
Are there any reasons to start path with a slash?
```suggestion
<script type="text/javascript" src="{% static 'admin/js/collapse.js' %}"></script>
```

Reply:
no, you're right. I fixed all you said.
Thank you for this suggestions